src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1560,10 +1561,24 @@ public ServiceStartupType StartupType
                 startupType = value;
             }
         }
+
         // We set the initial value to an invalid value so that we can
         // distinguish when this is and is not set.
         internal ServiceStartupType startupType = ServiceStartupType.InvalidValue;
 
+        /// <summary>
+        /// The following is the definition of the input parameter "SecurityDescriptorSddl".
+        /// Sets the SecurityDescriptorSddl of the service using a SDDL string.
+        /// </summary>
+        [Parameter]
+        [Alias("sd")]
+        [ValidateNotNullOrEmpty]
+        public string SecurityDescriptorSddl
```

Comment:
`SecurityDescriptorSddl` is rather redundant.  Could it be just `Sddl`?  cc @SteveL-MSFT

Reply:
The same is used for `Enable/Set/Register-PSSessionConfiguration` cmdlets. hence used this name as per @PaulHigin 's suggestion.

https://github.com/PowerShell/PowerShell/pull/8626#discussion_r250320136

Reply:
I agree it's redundant, but since there's precedent already it's better to be consistent